scripts/build/cht-core.yml.template

```diff
@@ -81,7 +81,7 @@ services:
       - "${NGINX_HTTP_PORT:-80}:80"
       - "${NGINX_HTTPS_PORT:-443}:443"
     volumes:
-      - cht-ssl:/root/.acme.sh/
+      - cht-ssl:"${SSL_VOLUME_MOUNT_PATH:-/root/.acme.sh/}"
```

Comment:
I had to remove the quotes around the value, I got this error: 
```
invalid mount config for type "volume": invalid mount path: '"/etc/nginx/private/"' mount path must be absolute
```

Reply:
yike! good catch - thanks. Fixed in 90bcda4ed

I'd caught this on the compose file I'd mocked up, but not on the template I'd committed. Is there an easy way to render the `*.yml.template` -> `*.yml` to test the templates being converted to real compose files?